portfolio/src/main/webapp/script.js

```diff
@@ -106,3 +106,49 @@ function getAboutMe(aboutMeId) {
     subContainer.style.display = 'none';
   }
 }
+
+/** Fetches comments from the server and add them to the DOM. */
+async function loadComments() {
+    const response = await fetch('/data');
+    const comments = await response.json();
+    console.log(comments);
```

Comment:
remove debugging logs before you submit, unless it's intentionally kept for production logging purposes.